### PR TITLE
feat: implement authenticated comment tests

### LINKED ISSUE #709: [Feature]: add authenticated e2e tests
### Description

Add user setup step to test authenticated actions
- test comments

### Explanation

_No response_

### PR DESCRIPTION
**Pull request by billc-dev**
## Description

- Add setup authenticated user and default to it
- Added tests to create, edit, delete, and share comment

## Motivation and Context

fixes #709

### CODE REVIEW

File: apps/web/playwright.config.ts [hunk 1]
```diff
@@ -6,24 +6,32 @@ export default defineConfig({
...
-  fullyParallel: true,
+  fullyParallel: false,
   forbidOnly: Boolean(process.env.CI),
   retries: process.env.CI ? 2 : 0,
   workers: process.env.CI ? 1 : undefined,
   reporter: [['html', { open: 'never' }]],
   use: {
-    baseURL: 'http://127.0.0.1:3000',
-    trace: 'on-first-retry',
+    baseURL: 'http://localhost:3000',
```
Comment (odicho) [inline apps/web/playwright.config.ts:15]: What is the reason for `127.0.0.1` -> `localhost`? I am just curious, I don't have much experience with Playwright!

**Comment by bautistaaa** [comment]
is this ready to ship

File: apps/web/playwright.config.ts (hunk 1 above)
Comment (bautistaaa) [reply apps/web/playwright.config.ts:15]: these mean the same thing right @billc-dev ?

Comment (billc-dev) [reply apps/web/playwright.config.ts:15]: They point to the same thing. I was just having trouble adding the cookie to `127.0.0.1`

**Comment by billc-dev** [comment]
@bautistaaa It's ready to ship but there's another PR that modifies comments and I want to see if the tests pass when that is merged.

**Review by bautistaaa — APPROVED** [review]

**Comment by bautistaaa** [comment]
idk why i cant update your branches. update branch and it should auto merge

**Pull request merged**